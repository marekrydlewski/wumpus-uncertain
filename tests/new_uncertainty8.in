10 10
0.25
??????????
??????????
B?????????
OB????????
OOB???????
OO?B??????
OOOOB?????
OOOB??????
OOB???????
OOOB??????
