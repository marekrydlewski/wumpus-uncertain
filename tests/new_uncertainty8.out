0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 
0.43 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 
0.00 0.75 0.25 0.25 0.25 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.57 0.25 0.25 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 0.68 0.25 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 0.00 0.62 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 0.00 0.00 0.37 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 0.00 0.37 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 1.00 0.25 0.25 0.25 0.25 0.25 0.25 
0.00 0.00 0.00 0.00 0.25 0.25 0.25 0.25 0.25 0.25 
